### DISCUSSION TITLE
how to do a nomination pool?

### DISCUSSION THREAD

**Discussion by lucasportella** [availproject/avail#620] [General]
got running the exemple next js project that shows how to do a send, tried to change the line 
`const tx = api.tx.dataAvailability.submitData("0x123456")`
to
`const tx = api.tx.nominationPools.join(100, 5)`

the method "join" has been working in my other app with polkadot and kusama(although we use the ApiPromise from polkadot lib)

the error I get is:
![image](https://github.com/user-attachments/assets/18d0b022-426d-4c6f-a104-dd573176f682)

am I missing something or there is a bug?

**Comment by Leouarz** [comment]
Hey, you can check for the future what custom errors mean by looking at the Wiki : https://github.com/availproject/avail/wiki/Transaction-Custom-IDs

It means you are using an appId different than 0
You probably created one and tried to use it for pool creation but App Ids different than 0 are only used for data submission.
With the correct parameters and app id 0, you should be able to create your pool.
